<!--
This is for including Chapter 1.  Notice that it's also good practice to name your chunk.  This will help you debug potential issues as you knit.  The chunk above is called intro and the one below is called chapter1.  Feel free to change the name of the Rmd file as you wish, but don't forget to change it here from chap1.Rmd. 
-->

<!--
The {#rmd-basics} text after the chapter declaration will allow us to link throughout the document back to the beginning of Chapter 1.  These labels will automatically be generated (if not specified) by changing the spaces to hyphens and capital letters to lowercase.  Look for the reference to this label at the beginning of Chapter 2.
-->

# Materials and Methods {#mat-met}

## Initial Isolation
Samples were taken from the primary filter and surfaces of the Reed Research Reactor pool using sterile swabs and counting water on the way to the pool floor as part of sample collection. Swabs were then used to grow lawns on nutrient broth agar plates (Sigma) and isolated through quadrant plating. A total of 10 isolates were grown from the samples, but only strains 1$^{\circ}$F5, 1$^{\circ}$FH2, and CM1 were used for comprehensive growth and UV analysis.

## UV testing of isolates

The bacterial strains and plasmids used in this study are listed in Table 1. Liquid cultures were grown in lysogeny broth (LB: Tryptone 10 g/L, Sodium Chloride 10 g/L, Yeast Extract 5 g/L) at 37$^{\circ}$C and plated on LB agar plates for subsequent assays before another overnight growth.  

<!-- insert table for isolates and naming conventions -->

The initial UV testing of the original 10 isolates from Spring 2021 and two controls—D. rad and DH5-$\alpha$— to compare phenotypes of these strains with D. Dev’s initial results. Using an inoculating loop, overnight cultures were streaked across the plate. Vertical quadrants of the plates were exposed for 0, 15, 30, or 60 seconds to 302 nm UV light using a 2 UV Transilluminator.  Aluminum foil was used to shield unexposed or less-exposed areas. After exposure, plates were incubated overnight at 30$^{\circ}$C and plate counts were performed.  

The secondary use of UV exposure was with serial dilutions of $10^0$, $10^{-1}$, $10^{-2}$, $10^{-3}$, $10^{-4}$, $10^{-5}$, and $10^{-6}$ to observe percent survival of each 1$^{\circ}$F5, 1$^{\circ}$FH2, CM1, D. rad, and DH5-$\alpha$. The dilutions were made using 1:10 ratios from $10^0$ to $10^{-5}$ in 1X Phosphate-Buffered Saline (PBS) (Sodium Chloride 8 g/L, Potassium Chloride 0.2 g/L, Sodium Phosphate Dibasic 1.44 g/L, Potassium Phosphate Monobasic 0.245 g/L, pH 7.5 $\pm$0.2 at 25$^{\circ}$C). Then, 10 $\mu$L from each dilution was spotted onto plates. Sets of three plates were exposed to UV for 0, 30, and 60 seconds gel-side-down, then incubated overnight at 30$^{\circ}$C. After incubation, colonies were counted and CFU/mL from serial dilutions was used to calculate percent survival across the different exposure times.  

The third set of UV exposures was performed with normalized $10^0$ CFU/mL across all strains for better comparison of percent survival and growth rate of strains. Normalization was performed by growing overnights of each strain, plating serial dilutions, incubating overnight, then counting CFU/mL. The CFU/mL counted from the first plating was then used to dilute fresh overnight liquid culture to the same CFU/mL to start for serial dilutions with UV exposure. 

## 16S PCR Analysis

The following 16S primers were used to determine genus level classification of isolates. Prior to PCR, isolates were incubated overnight at 30$^{\circ}$C in nutrient broth.

<!-- insert table for primers -->
<!-- insert table for primer pairs -->

PCR was conducted to amplify 16S rRNA to assess microbial diversity using primers listed in Table 2 and the combinations in Table 3 using Q5 High-Fidelity DNA Polymerase (New England BioLabs, Ipswich, MA) according to standard procedures. Cycling conditions (T100, BioRad) were 98$^{\circ}$C 40 sec, followed by varied cycles of 95$^{\circ}$C 10 sec, annealing temperature 55$^{\circ}$C 30 sec, 72$^{\circ}$C 30 sec. Samples were run in duplicate for 30 cycles total. 

DNA samples were separated on 25 mL 0.8% agarose (GenePure) in 1xTBE with 3 $\mu$L SybrSafe at 90-120 V. 12 $\mu$L of PCR product was prepared with 3 $\mu$L volume 6X DNA gel loading mix and run next to 250 ng of Quick-Load Purple 2-Log DNA ladder (0.1-10.0 kb). For controls, nuclease-free water was run in parallel. 

QIAquick Gel Extraction Kit (Qiagen Corp., Carlsbad, CA) was used to extract DNA from PCR products run on agarose gels to send in for sequencing of segments between 16S primer pairs. DNA fragments were excised as bands from agarose gel using scalpel on Transilluminator for visibility, and extraction was performed according to standard procedures. Product was sent to ACGT, Inc. (Wheeling, IL) for sequencing, and resulting FASTA files were analyzed via SnapGene Viewer ( GSL Biotech LLC, San Diego, CA) and run through NCBI nucleotide blasts for genus-level identification.

## Growth Rate Analysis  

To observe different growth rates independent of radioresistance, turbidity was measured by an Infinite 200 Pro microplate reader (Tecan, Männedorf, Switzerland) at 600 nm absorbance. Bacteria grown in LB were incubated at 30$^{\circ}$C for 24 hours at 30 minute intervals in a Greiner 96 Flat Bottom Transparent Polystyrene well plate.

## Gram Staining

Specimen was heat-fixed onto microscope slide by adding one drop of dH2O to slide and transferring minute amount of color on solid media onto slide. If in liquid media, one drop was transferred via inoculating loop. Mixture was spread evenly across 1.5 cm diameter circle and air dried. Using clothespin, slide was passed over a gentle flame to fix the cells.  5 drops of crystal violet stain was applied over fixed culture and incubated for 60 seconds before rinsing with dH2O. Then, 5 drops of iodine solution were added, and incubated for 30 second before rinsing with dH2O. A couple drops of 95% ethanol  was used to decolorize for no more than 5 seconds before rinsing with dH2O until solution ran clear. Slide was counterstained with 5 drops of safranin solution for 20 seconds and rinsed with dH2O before blotting dry to observe under microscope.

## DNA Sequencing Analysis

All plasmid constructs were confirmed to be correct by DNA sequencing analysis performed at the Vollum Institute at the Oregon Health and Science University.



## Additional resources

- _Markdown_ Cheatsheet - <https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet>

- _R Markdown_
    - Reference Guide - <https://www.rstudio.com/wp-content/uploads/2015/03/rmarkdown-reference.pdf>
    - Cheatsheet - <https://github.com/rstudio/cheatsheets/raw/master/rmarkdown-2.0.pdf>

- _RStudio IDE_
    - Cheatsheet - <https://github.com/rstudio/cheatsheets/raw/master/rstudio-ide.pdf>
    - Official website - <https://rstudio.com/products/rstudio/>

- Introduction to `dplyr` - <https://cran.rstudio.com/web/packages/dplyr/vignettes/dplyr.html>

- `ggplot2`
    - Documentation - <https://ggplot2.tidyverse.org/>
    - Cheatsheet - <https://github.com/rstudio/cheatsheets/raw/master/data-visualization-2.1.pdf>

