```{r include_packages_4, include = FALSE}
# This chunk ensures that the thesisdown package is
# installed and loaded. This thesisdown package includes
# the template files for the thesis and also two functions
# used for labeling and referencing
if (!require(remotes)) {
  if (params$`Install needed packages for {thesisdown}`) {
    install.packages("remotes", repos = "https://cran.rstudio.com")
  } else {
    stop(
      paste(
        'You need to run install.packages("remotes")',
        "first in the Console."
      )
    )
  }
}
if (!require(dplyr)) {
  if (params$`Install needed packages for {thesisdown}`) {
    install.packages("dplyr", repos = "https://cran.rstudio.com")
  } else {
    stop(
      paste(
        'You need to run install.packages("dplyr")',
        "first in the Console."
      )
    )
  }
}
if (!require(ggplot2)) {
  if (params$`Install needed packages for {thesisdown}`) {
    install.packages("ggplot2", repos = "https://cran.rstudio.com")
  } else {
    stop(
      paste(
        'You need to run install.packages("ggplot2")',
        "first in the Console."
      )
    )
  }
}
if (!require(bookdown)) {
  if (params$`Install needed packages for {thesisdown}`) {
    install.packages("bookdown", repos = "https://cran.rstudio.com")
  } else {
    stop(
      paste(
        'You need to run install.packages("bookdown")',
        "first in the Console."
      )
    )
  }
}
if (!require(thesisdown)) {
  if (params$`Install needed packages for {thesisdown}`) {
    remotes::install_github("ismayc/thesisdown")
  } else {
    stop(
      paste(
        "You need to run",
        'remotes::install_github("ismayc/thesisdown")',
        "first in the Console."
      )
    )
  }
}
library(thesisdown)
library(dplyr)
library(ggplot2)
library(knitr)
flights <- read.csv("data/flights.csv", stringsAsFactors = FALSE)
```


# Conclusion {#conclusion}
In this study, I explored organisms living in extreme environments, such as the water systems of nuclear reactors, and the possibility of observing their genome for unique methods of resistance to ionizing radiation, whether it be through mechanisms of protection against radiation-induced oxidative damage to proteins, or repair of such damage. Strain CM1 of Priestia Megaterium is a robust bacterium isolated from surfaces surrounding the core of the Reed Research Reactor. Of the 10 strains isolated from various areas of the RRR water system, only CM1 was radioresistant enough to portray the same level of growth as Deinococcus radiodurans when calculating survivability and growth rates. The radioresistance of CM1 was tested by UV light against controls DH5-a and Deinococcus radiodurans using serial dilution and growth assays to assess survivability and growth rates affected by exposure to UV light. Identification on a genus based level was performed via 16S analysis and sequencing by ACGT (ACGT, Inc., Wheeling, IL), and its genome was sequenced by Novogene (Novogene Corporation Inc., Sacramento, CA). 
The results of my experiments have laid groundwork for future studies with this strain of radioresistant bacteria. Not only have we found more information on species on mutations from the reference genome, but also the specific genes that have changed. _From the BLANK gene mutations observed, function of the produced proteins can be experimentally tested under radioactive stress. In addition to that, more samples of this bacteria can be isolated from period samples of RRR water to observe mutations over time near the core of a nuclear reactor. [[MORE ABOUT GENOME ANALYSIS RESULTS]] While the mechanisms of survival have not been elucidated in this study, sequencing the genome allows for future studies such as protein knock-out assays that can help figure out which proteins or mechanisms are responsible for the survivability of this bacteria. This work has already begun through the study of the ATP-dependent RecD-like DNA helicase of CM1 (N. Xu and H. Monyatovsky, unpublished data)._
Search for more efficient methods of radioactive waste cleanup has been in progress since the 1900s with the nuclear industry boom [@ashley_review_1989]. Both bacteria and fungi have been studied for bioabsorption and remediation of radionuclides through accumulation of contaminants in life and death. Biopolymers and biosorption using biomass have great potential for not just nuclear waste treatment, but waste treatment in general. In more recent publications than a paper from 1989, an increase in uranium contamination as chemical toxicity and radioactivity is slowly creeping its way from power plants and uranium mills to the top of the food chain in the form of oxides, precipitates, complexes, and natural minerals. Researchers have observed uptake of uranium oxide through biosorption and biomineralization by Bacillus subtilis ATCC-6633 [@song_nonreductive_2019,] and the environmental conditions for bioreduction by numerous other microorganisms [@wufuer_uranium_2017]. Radioresistant bacteria are constantly being discovered, including at least 100 from the Taklimakan Desert in a paper published March 2022 [@liu_high_2022]. 
With new textbooks, research, and articles being published on bioremediation of radioactive waste, there’s a long way to go for efficient usage of these materials. With the work done in this study, I hope that one day it can contribute to creating consortiums of bacteria used to take care of mixed wastes, to combine with the research of plastic degradation, chemical wastes, and biological waste. Looking for bacteria in harsh environments such as nuclear reactor facilities or the hottest deserts in the world can help with the extreme conditions we’ve created for Earth. Just as scientists have searched for bacteriophages in hospital sewage as cures for bacterial infections [@lin_phage_2017], just as the Toxic Jungle cleanses the air for the Valley of the Wind [@miyazaki_nausicaa_2004], working with tools given by the environment can be key to solving many of the world’s waste crises.   
